src/typography/social/index.md

```diff
@@ -28,62 +23,59 @@ Whilst there are many ways to build visual hierarchy, mixing weight and scale ac
 Indicative examples for illustrative purposes only.
 {% endcallout %}
 
-{% sectionHighlight { classes: "app-section-highlight--light-blue" } %}
-
 ![Graphic promoting pension information, with labels pointing out different type styles for the tag, headline and body copy, each with different font sizes and weights.](./type-hierarchy.png)
 
-{% endsectionHighlight %}
-
 ## Styles
 
 Consistent use of type styles aids clarity and hierarchy.
 
 Headings should be attention-grabbing, whilst body text should prioritize readability with appropriate line spacing and contrast.
 
-<!-- TODO: lots to do here, some of the below should probably be in images -->
-
 {% grid { columns: { mobile: 2, desktop: 2 } } %}
 
-<div class="app-top-border">
+{% gridCell {classes: "app-top-border"} %}
 
 ### Headline styles
 
 Bold, over a maximum 5 lines
 Light, over a maximum 5 lines
 Any content over 5 lines should be formatted as a body style
 
-</div>
-<div>
-img goes here
-</div>
+{% endgridCell %}
+{% gridCell %}
 
-<div class="app-top-border">
+![](./headline-styles.svg) ![](./headline-styles-light.svg)
```

Comment:
The added whitespace on these new images looks especially bad in this context because it is in the middle of text in the middle of the page.